cmd/controller/app/controller.go

```diff
@@ -65,7 +66,7 @@ func Run(kubeconfigPath string,
 	}
 
 	// start the controller
-	if err := newController.Start(); err != nil {
+	if err := newController.Start(context.TODO()); err != nil {
```

Comment:
Are we sure that the correct thing to do here is use `context.TODO()`? From https://pkg.go.dev/context#Background it seems to me like `context.Background()` fits better. But it might be that I'm misunderstanding the use of `context.TODO()`.
@liranbg WDYT?

(Same question for all `nuctl` instances and in the tests)

Reply:
@TomerShor  - @quaark is right. when you _initial_ the first context in the flow, you use `.Background` as suggested by context's stdlib documentation.
You use `.TODO()` when you were not able to get the `ctx` from your callee